ql/pricingengines/blackformula.hpp

```diff
@@ -235,6 +236,27 @@ namespace QuantLib {
                         Real stdDev,
                         Real displacement = 0.0);
 
+/*! Black 1976 probability of being in the money in the asset martingale
+        measure, i.e. N(d1).
+        It is a risk-neutral probability, not the real world one.
+    */
+    Real blackFormulaItmAssetProbability(
```

Comment:
Following the pattern of the existing ones, it should be `blackFormulaAssetItmProbability`.

Reply:
Up to you, I was inspired by naming of `BlackCalculator::itmAssetProbability()`

Reply:
Let's be consistent at least in the same file.

Reply:
Cahnged to `blackFormulaAssetItmProbability` and `bachelierBlackFormulaAssetItmProbability`